enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/EditorButton.module.css

```diff
@@ -0,0 +1,14 @@
+.editorButton:not([data-css-specificity-hack="😅"]) {
```

Comment:
ActionIcon is taking control over these styles, so the data specificity hack is needed.